test/integration/node_unique_views_test.rb

```diff
@@ -7,14 +7,14 @@ class NodeInsertExtrasTest < ActionDispatch::IntegrationTest
     assert_equal 0, Impression.count
 
     assert_difference 'Impression.count', 1 do
-      get "wiki/#{nodes(:about).slug}"
+      get "/wiki/#{nodes(:about).slug}" #gave the complete address changed from wiki to /wiki
```

Comment:
feel free to remove this comment anytime

Reply:
Yeah @Gauravano I will remove these